pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.poetry]
+name = "npc-gzip"
+version = "0.1.0"
+description = "Code for Paper: “Low-Resource” Text Classification: A Parameter-Free Classification Method with Compressors"
+authors = [
+    "Gin Jiang <bazingagin@gmail.com>",
+    "Zach Bloss <zacharybloss@gmail.com>",
+]
+maintainers = [
+    "Zach Bloss <zacharybloss@gmail.com>",
+    "Gin Jiang <bazingagin@gmail.com>",
+]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/bazingagin/npc_gzip"
+repository = "https://github.com/bazingagin/npc_gzip"
+keywords = ["npc_gzip", "knn-gzip"]
+packages = [{include = "npc_gzip"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+numpy = "^1.25.1"
```

Comment:
This is bumped in `poetry.lock` as of a08a9dd, but I suggest bringing it up to `"^1.25.2"` here in `pyproject.toml` as well, unless 1.25.1 has been intentionally chosen as the minimum version. (If there were more such changes to make, [`poetry up`](https://github.com/MousaZeidBaker/poetry-plugin-up) could help, but I think this is the only one.)